okex-client/tests/client.rs

```diff
@@ -154,80 +156,71 @@ async fn transfer_state() -> anyhow::Result<()> {
 
 #[tokio::test]
 async fn place_order() -> anyhow::Result<()> {
-    if let (Ok(_), Ok(_), Ok(_)) = (
-        env::var("OKEX_DEMO_API_KEY"),
-        env::var("OKEX_DEMO_SECRET_KEY"),
-        env::var("OKEX_PASSPHRASE"),
-    ) {
-        let client = demo_okex_client();
-
-        let order_id = client
-            .place_order(
-                "BTC-USD-SWAP".to_string(),
-                "cross".to_string(),
-                "buy".to_string(),
-                "long".to_string(),
-                "market".to_string(),
-                1,
-            )
-            .await?;
+    std::env::set_var("OKEX_SIMULATED_TRADING", "1");
+
+    let client = demo_okex_client();
+    let instrument = OkexInstrumentId::BtcUsdSwap;
+    let margin = OkexMarginMode::Cross;
+    let position_side = OkexPositionSide::long();
```

Comment:
don't use the method, use the `::Long` syntax... get around the string issue by implementing `Display`